python/lsst/ip/diffim/dcrModel.py

```diff
@@ -395,6 +395,9 @@ def buildMatchedExposure(self, exposure=None,
         maskedImage.image = templateImage[bbox]
         maskedImage.mask = self.mask[bbox]
         maskedImage.variance = self.variance[bbox]
+        # The variance of the stacked image will be `dcrNumSubfilters`
+        #   times the variance of the individual subfilters.
```

Comment:
The extra spaces in this second comment line seem, well, extra. But the explainer is welcome!